interceptors/logging/logging.go

```diff
@@ -14,10 +14,20 @@ import (
 	"github.com/grpc-ecosystem/go-grpc-middleware/v2/interceptors"
 )
 
+// ENUM values for enabling one of the three decisions for logging.
+type Decision int
```

Comment:
Comment has to start with `Decision` here

Reply:
Done